docs/loading-in-profiles.md

```diff
@@ -21,16 +21,77 @@ uploadprofile /path/to/profile.js
 
 > `https://perf-html.io/from-url/{URL}`
 
-Profiles can also be loaded in from arbitrary URLs. In addition, you can then upload it to the online storage directly from perf.html's interface.
+Profiles can also be loaded in from arbitrary URLs. In addition, you can then upload it to the online storage directly from perf.html's interface. The profile needs to be served from https, and have the appropriate `Access-Control-Allow-Origin` header set to either `*` or `https://perf-html.io`.
 
 See the function below for an easy utility for converting to the proper URL format. Here is a simple utility function to turn a profile URL into a perf.html URL to view it:
 
 ```js
 function getPerfHtmlUrl (profileUrl) {
-  return `https://perf-html.io/from-url/${encodeURIComponent(profileUrl)}`;
+  return "https://perf-html.io/from-url/" + encodeURIComponent(profileUrl)};
 }
 ```
 
+Or in Python:
+
+```python
+import urllib
+
+def get_perf_html_url(url):
+    return "https://perf-html.io/from-url/" + urllib.quote(url, safe="")
+```
+
+Here is an example profile server written in Node.js:
```

Comment:
wondering if we couldn't direct users to https://www.npmjs.com/package/http-server ?
It seems to support cors, https.

alternatively we could export this code to our own npm server that would use that package.

Reply:
Mozilla automation is primarily python, so I'm not sure if it'd be worth supporting a server ourselves. I mainly wrote the code as a basic example to get it working on my machine, and it seemed valuable for the docs.